api/app/wildfire_one/wfwx_api.py

```diff
@@ -334,3 +364,18 @@ async def get_fire_centers(session: ClientSession, header: dict,) -> List[FireCe
     """ Get the fire centers from WFWX. """
     wfwx_fire_centers = await get_station_data(session, header, mapper=fire_center_mapper)
     return list(wfwx_fire_centers.values())
+
+
+async def get_dailies_for_stations_and_date(session: ClientSession,
+                                            header: dict,
+                                            time_of_interest: datetime,
+                                            unique_station_codes: List[int],
+                                            mapper=yesterday_dailies_list_mapper):
```

Comment:
Just curious why the mapper is included as a parameter? Do you anticipate other callers of this function that will need a different mapper?

Reply:
There's a lot of duplication in `wfwx_api`, there's `get_dailies_for_stations_and_date`, `get_daily_actuals_for_stations_between_dates` that use `get_dailies_generator` and `get_raw_dailies_in_range_generator`. Eventually I want to reduce that duplication and they generally differ on what they extract (or map) from wfwx, so I use a parameterized mapper. There's a few other tricky bits I haven't figured out yet so the duplication still exists and I don't want to break anything in this PR, just want to shape it towards less duplication.

Reply:
Thanks for the explanation!